github/editor.go

```diff
@@ -52,12 +54,18 @@ type Editor struct {
 	openEditor func(program, file string) error
 }
 
-func (e *Editor) AddCommentedSection(text string) {
-	startRegexp := regexp.MustCompilePOSIX("^")
-	endRegexp := regexp.MustCompilePOSIX(" +$")
-	commentedText := startRegexp.ReplaceAllString(text, e.CS+" ")
-	commentedText = endRegexp.ReplaceAllString(commentedText, "")
-	e.Message = e.Message + "\n" + commentedText
+func (e *Editor) AddCommentedSections(sections []string) {
```

Comment:
Could this method signature stay the same as before, but only add the scissors block on its first invocation and not in subsequent ones?